MAJ docs pour v1.0

SpaceFox opened the pull request:
| Q | R |
| --- | --- |
| Correction de bugs ? | Non |
| Nouvelle Fonctionnalité ? | Non |
| Tickets concernés | #1404 (mais partiellement) |

poulp committed:
ajout de tests pour les formulaires de mp

poulp committed:
fonction get_current_user mise en paramètre optionnel dans les modèles

car non utilisable dans les tests et ne change pas le reste du code

poulp committed:
verification du user en paramètre optionnel

si None alors get_current_user

poulp committed:
la requête pour récuperer le dernier message non lu renvoie une exception

du coup la fonction qui devait renvoyer le premier message non lu, renvoyait le premier message du topic

poulp committed:
ajout des tests pour les modèles des mp

poulp committed:
vérification qu'un utilisateur appartient bien à un topic avant de suppr

poulp committed:
ajout de tests pour les views mp

poulp committed:
empeche un utilisateur de s'envoyer un mp tout seul

poulp committed:
ajout de tests pour les views

Eskimon committed:
Autopopuplate du field username pour le form de connexion

Eskimon committed:
Form de connexion directement dispo apres validation du compte

Le formulaire de connexion est dorénavant directement disponible
lorsque un nouveau membre vient valider son compte.
Cela évite un chargement de page supplémentaire pour aller se connecter.
Le champ "username" est alors pré-remplie avec ce nouvel utilisateur.

Ge0 committed:
More information into confirmation mails

poulp committed:
ajout de tests pour les vues answer, edit_post, leave et add_participants

plus correction des bugs trouvés

poulp committed:
clean code, pep8

AlexandreBroudin committed:
Suppression du lien vers Google Chrome Frame

Google Chrome Frame n’existe plus, il n’a donc plus raison d’être
present sur le site.
https://www.google.com/chromeframe/about/
Aussi je change le test lt IE 7 vers lt IE 9, IE 8 étant en fin de vie
http://theie8countdown.com/

Eskimon committed:
Merge pull request #1100 from AlexandreBroudin/dev

Suppression du lien vers Google Chrome Frame

Ge0 committed:
Fixup confirmation mail

Ge0 committed:
Fixup html template

Eskimon committed:
Merge pull request #1075 from zestedesavoir/mail_template

More information into confirmation mails

coma94 committed:
fix formatage MP refus tuto (#1142)

Eskimon committed:
Merge pull request #1145 from coma94/fix-1142

Corrige le formatage du MP d'un refus de tuto (#1142)

Eskimon committed:
Merge pull request #1082 from poulp/tests-mp

Ajout de tests pour les mp et correction de bugs

Commit:
Règle le problème des blocs oranges intro/conclu lorsque le tuto est en bêta

Commit:
Enlève les fichier ~

devock committed:
Merge pull request #1185 from Florianboux/issue-1182

Fix #1182

coma94 committed:
Auto-remplissage de l'url d'origine d'un tuto #1201

Eskimon committed:
Merge pull request #1204 from coma94/fix-1201

Auto-remplissage de l'url d'origine d'un tuto #1201

dralliw committed:
Merge pull request #1255 from zestedesavoir/master

copie sur la branche prod

dralliw committed:
Merge pull request #1263 from zestedesavoir/master

MEP Release

Commit:
creation auto de sujet pour les tuto beta

Alex-D committed:
Merge branch 'master' into dev

Conflicts:
	templates/base.html
	templates/tutorial/part/view.html
	templates/tutorial/tutorial/view.html
	zds/mp/tests.py
	zds/tutorial/views.py

Alex-D committed:
Corrige un petit détail dans les ML

Alex-D committed:
Merge pull request #1316 from Alex-D/hotfix_mp

Corrige un petit détail dans les ML

dralliw committed:
nouvelle quote

dralliw committed:
mise à jour d'une citation

Alex-D committed:
Merge pull request #1323 from firm1/features-quotes

nouvelle quote

Akulen committed:
Correction du mode preview de l'edition d'articles

Akulen committed:
Fix : licence optionnelle

SpaceFox committed:
Merge pull request #1355 from rigauxt/hotfix_article_preview

Correction du preview lors de l'edition d'articles

dralliw committed:
fichier inutile depuis un moment

Alex-D committed:
Merge pull request #1369 from firm1/nettoyage

Suppression de fichiers inutiles

Commit:
ajout des tops tags dans la top bar

dralliw committed:
correction de la page des forums

Commit:
correction de la page des catégories

sandhose committed:
Optimisation du PNG du sprite

Commit:
MAJ docs pour v1.0

Alex-D commented on CONTRIBUTING.md:
```diff
@@ -12,19 +12,26 @@ Les contributions externes sont les bienvenues !
+1. Si vous avez fait des modifications du _front_, jouez les tests associés : `gulp test`
+1. Si vous modifiez le modèle (les fichiers models.py), n'oubliez pas de créer les fichiers de migration : `python manage.py schemamigration app_name --auto`
+
 5. Poussez votre travail et faites une _pull request_
```
Mets un 1. là aussi du coup, non ?

pierre-24 replied:
Je serai d'avis que la liste reste lisible sans être formatée, au cas ou, donc avoir les "bons" numéros. Mais c'est purement subjectif.

Taluu replied:
+:100:

Alex-D commented on doc/workflow.md:
```diff
@@ -0,0 +1,75 @@
+Cette page d�taille le _workflow_ utilis� sur Zeste de Savoir. Elle est l� surtout pour satisfaire votre curiosit�, � moins d'avoir les droits de faire une Mise En Production (MEP). La [page de contribution](CONTRIBUTING.md) devrait r�pondre � vos questions quant au processus de d�veloppement.
```
Il y a des soucis d'encodage sur tout ce fichier.

sandhose committed:
Fix l'autocompletion des pseudos avec espaces

sandhose committed:
Update des dépendances NPM

Nicofuma commented:
A des moments tu dis "`dev` ne devrait contenir que des merge de PR, aucun commit direct." mais en fait vaut mieux qu'aucune branche ne contienne aucun commit direct (ni dev, ni release, ni prod) comme ça : 
- tu as l'assurance que au moins une personne a revu le fix avant qu'il ne soit fusionné (y compris les hot fix)
- le revert est plus facile (il suffit de revert le merge commit, ça implique par contre de toujours faire des merge en --no-ff)
- ça permet de sortir des stats plus facillement si on veut :) (--depth=1 te sortira la liste des merge et donc des PR)

Sinon, ne jamais remonter release dans dev c'est ce qu'il y a de plus simple d'un point de vue ressources en effet, faut juste espérer que le code restera de qualité :)

Nodraak committed:
fix issue #1330

Eskimon committed:
Merge pull request #1386 from firm1/feature-top-tags

Ajout des tops tags dans la top bar

Eskimon committed:
Changement de methode d'autoremplissage du LoginForm

Eskimon committed:
Bulletin d'adhesion plus souple #1079

Eskimon committed:
Adhesion ajout date de naissance #1349

Nodraak committed:
Doc pour les parametres obscurs de la fonction send_mp()

Nodraak committed:
Envoie d'un msg pr l'ajout/suppression d'un auteur

Eskimon committed:
MAJ du test du form adhesion

Commit:
initialisation de la documentation

Commit:
mise à jour du gitignore

Commit:
mise à jour du gitignore

Commit:
suppression des doublons

dralliw committed:
typo

dralliw committed:
typo

Commit:
import d'une archive d'images dans une galérie

Commit:
Merge branch 'master' into prod

Commit:
Merge branch 'master' into dev

Conflicts:
	quotes.txt

Commit:
amélioration des flux rss

Commit:
mise à niveau des versions

dralliw committed:
typo

Commit:
controle de l'extention du fichier

Commit:
Issue #1416: Tests cassés lors du merge dans dev

Commit:
conflits

SpaceFox committed:
Merge pull request #1417 from SpaceFox/issue_1416_tests_sur_dev

Issue #1416: Tests cassés lors du merge dans dev

Commit:
Merge branch 'dev' of https://github.com/zestedesavoir/zds-site into fix-preview-comments

Commit:
ajout des messages précédents lors de la preview

Commit:
correction des tests

Commit:
Merge branch 'dev' of https://github.com/zestedesavoir/zds-site into dev

Commit:
mp et reservation lors d'une maj de contenu en validation

Commit:
correction

dralliw committed:
mise à jour des tests

dralliw committed:
correction du test de validation

dralliw committed:
mise à jours des vérifications

dralliw committed:
test de l'arrivée d'un MP au staff

Eskimon committed:
Suppression d'un lien de connexion redondant

Alex-D committed:
Merge pull request #1401 from Nodraak/fix_#1330

Vide le formulaire d'adhésion à l'association une fois envoyé

dralliw committed:
corrige les problème d'encodage de formulaire

SpaceFox committed:
Merge pull request #1403 from Eskimon/evo_990

Connexion rapide suite à la validation du compte à l'inscription

Eskimon committed:
Merge pull request #1428 from firm1/update_profile

Edition du profil d'un membre par le staff

Alex-D committed:
Merge pull request #1398 from sandhose/npm-update

Update des dépendances NPM

Commit:
correction de la detection trop souple

Alex-D committed:
Rend le test travis sur les assets plus complet

Alex-D committed:
Change les sources des images pour minitures

Alex-D committed:
Ajoute un code d'insertion : miniture + lien vers maxi

Alex-D committed:
Stylise correctement le code inline dans es listes

Alex-D committed:
Augmente la taille du textarea de signalement

Alex-D committed:
Amélioration du rendu des alertes

Alex-D committed:
Corrige une incohérence ordi/mobile

Alex-D committed:
Correction des labels du formulaire de connexion

Alex-D committed:
Retire les astérisques inutiles sur le formulaire de connexion

Alex-D committed:
Supprime les boutons annuler inutiles et renomme les validation

Alex-D committed:
Boutons soumission vert au lieu de bleu + factorisation

Alex-D committed:
Uniformise les nom 'Mon profil'

Alex-D committed:
Merge pull request #1433 from Alex-D/optim_tests_front

Rend le test travis sur les assets plus complet

Alex-D committed:
Change image en img

Alex-D committed:
Ajoute l'extension mhchem a MathJax

Alex-D committed:
Spécifie les dimentions de Clem

Alex-D committed:
Optimize Modernizr and JS loading

SpaceFox committed:
Merge pull request #1441 from Alex-D/add-mhchem

Ajoute l'extension mhchem à MathJax

Alex-D committed:
Retire 'les modifications' des boutons

Alex-D committed:
Merge pull request #1440 from Alex-D/optim-ux-navigation

[UX] Uniformise les 'Mon profil'

Alex-D committed:
Merge pull request #1406 from Eskimon/evo_1079_1349

Amelioration du formulaire d'adhesion

Alex-D committed:
Merge pull request #1396 from sandhose/fix-opti-sprites

Optimisation du sprite

dralliw committed:
Typo

Nodraak committed:
ajout du lien vers les articles/tuto de l'auteur

dralliw committed:
typo

Alex-D committed:
Merge pull request #1420 from firm1/feature-improve-validation

Amélioration de la mise à jour en validation

Commit:
Security fix in PyYAML

Commit:
Merge branch 'hotfix/pyyaml' into dev

Alex-D committed:
Merge pull request #1442 from Alex-D/clem-uncut

Clem' n'est plus coupée sur Firefox \o/

Alex-D committed:
Merge pull request #1434 from Alex-D/optim-miniatures

Miniatures dans les galeries

Alex-D committed:
Merge pull request #1435 from Alex-D/fix-code-in-list

Stylise correctement le code inline dans les listes

SpaceFox committed:
Merge pull request #1449 from SpaceFox/hotfix/pyyaml

Hotfix/pyyaml

Alex-D committed:
Merge pull request #1438 from Alex-D/optim-ux-login

[UX] Amélioration de l'expérience utilisateur sur la page de connexion

Alex-D committed:
Merge pull request #1439 from Alex-D/optim-ux-btns

[UX] Optimisation de l'expérience utilisateur des boutons

Alex-D committed:
Merge pull request #1445 from Alex-D/optim-modernizr

Optimize Modernizr and JS loading

Alex-D committed:
Merge pull request #1437 from Alex-D/optim-alertes

Améliore le système de signalement des messages

pierre-24 commented:
On en est ou, ici ?

dralliw committed:
Typo

dralliw committed:
traduction

dralliw committed:
mise à jour de la doc linux

dralliw committed:
désactive l'indexation des pages off

Eskimon committed:
Merge pull request #1456 from firm1/unlink_tuto_off

Désactive l'indexation des pages offline

Nodraak committed:
merge conflit

Eskimon committed:
Merge pull request #1431 from firm1/fix-mp

Correction de la detection trop souple des pseudos dans les MPs

sandhose committed:
Merge pull request #1397 from sandhose/fix-autocomplete

Fix l'autocompletion des pseudos avec espaces

Commit:
factorisation des codes de flux forums

Commit:
edition du post principal et prevention auteur

Commit:
correction des liens dans les articles

Commit:
correction du flux par tag

Eskimon committed:
Merge pull request #1419 from firm1/fix-preview-comments

Aperçu des messages postés lors de la prévisualisation

Commit:
correction du message de maj

Eskimon committed:
Merge pull request #1414 from firm1/feature-improve-feed

Amélioration des flux RSS

Eskimon committed:
Merge pull request #1415 from firm1/beta-tuto-behavior

Amélioration de la mise en beta d'un tutoriel

Eskimon commented:
Re pop, c'est mergeable en fait ou pas ?

pierre-24 committed:
ZIP versionnées sur tutos (#1458 et #1460)

Nodraak committed:
url correct + lignes de 120 char. max

pierre-24 committed:
ZIP versionnées sur articles (#1458 et #1460)

Nodraak committed:
merge

pierre-24 committed:
Suppression de l'archive après téléchargement

pierre-24 committed:
Retrait d'une variable inutile

pierre-24 committed:
Retrait d'une variable inutile

Eskimon committed:
Merge pull request #1407 from Nodraak/fix_#1173_msg_new_author

Informe le nouvel auteur lors de  son ajout aux auteurs d'un tutoriel/article

pierre-24 committed:
ajout d'un test de sécurité

Seul l'auteur et les personnes possédant le droit
peuvent téléchargeer l'archive contenant la version
draft.

pierre-24 committed:
Correction du dossier temporaire

pierre-24 committed:
Amélioration de la condition sur la version online

pierre-24 committed:
Correction d'une permisssion

Situphen committed:
Typo

Eskimon committed:
Merge pull request #1464 from Situphen/dev

Correction orthographique

pierre-24 committed:
Ajout de tests unitaires

pierre-24 committed:
Merge remote-tracking branch 'upstream/dev' into fix_archive_1458_1460

pierre-24 committed:
Typo

Situphen committed:
Simplification d'une commande

Eskimon committed:
Merge pull request #1465 from Situphen/dev

Simplification d'une commande

Alex-D commented:
Il y a toujours des soucis d'encodage, ça doit être nettoyé au moins à ce niveau.

dralliw committed:
crée les répertoires s'ils n'existent pas

dralliw committed:
Merge pull request #1409 from firm1/add-docs

Initialisation de la documentation

Eskimon committed:
Merge pull request #1461 from pierre-24/fix_archive_1458_1460

Améliore le comportement des archives de tuto/article

dralliw committed:
limite l'import uniquement aux images

Eskimon committed:
Merge pull request #1410 from firm1/import-images-from-archives

Import d'une archive d'images dans une galerie

Alex-D committed:
Augmente la qualité du logo + effet hover

Alex-D committed:
Corrige le word-wrap

Alex-D committed:
Retire une condition inutile d'un template

Eskimon committed:
Merge pull request #1480 from Alex-D/fix-promote

Retire une condition inutile d'un template

sandhose committed:
Merge pull request #1478 from Alex-D/fix-logo

Augmente la qualité du logo + effet hover

sandhose committed:
Merge pull request #1479 from Alex-D/fix-word-wrap

Corrige le word-wrap

Commit:
Merge branch 'dev' of github.com:ZesteDeSavoir/zds-site into dev

SpaceFox commented:
Ah fuck elle était basée sur master celle-ci... j'en refais une sur dev.
